helper/sshutil/session_pool.go

```diff
@@ -94,7 +96,7 @@ type conn struct {
 // closes the ssh client
 func (c *conn) close() {
 	c.c.Close()
-	metrics.IncrCounter(metricsutil.CleanupMetricKey([]string{"ssh-connections-pool", "closes", c.name}), 1)
+	metrics.IncrCounter(metricsutil.CleanupMetricKey([]string{"ssh-connections-pool", "closes"}), 1)
```

Comment:
Why do not have connection name in labels here? 

```suggestion
	labels := []metrics.Label{metrics.Label{Name: "ConnectionName", Value: c.name}}
	metrics.IncrCounterWithLabels(metricsutil.CleanupMetricKey([]string{"ssh-connections-pool", "closes"}), 1, labels)
```

Reply:
Here we measure the number of closed ssh connections. Its not an information about a particular connection, that's why connection name is not pertinent

Reply:
ok